pkgs/development/libraries/libtiff/default.nix

```diff
@@ -1,36 +1,38 @@
-{ stdenv, fetchurl, pkgconfig, zlib, libjpeg, xz }:
+{ stdenv
+, fetchFromGitLab
+
+, pkgconfig
+, autogen
+, autoconf
+, automake
+, libtool
+
+, zlib
+, libjpeg
+, xz
+}:
 
-let
-  version = "4.0.9";
-in
 stdenv.mkDerivation rec {
+  version = "2018-11-04";
```

Comment:
```suggestion
  version = "unstable-2018-11-04";
```

Reply:
Good point!  Based on the guidelines, I put the `unstable` in the package name, not version, but :+1: